src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,379 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    public static final int DONE_START = 5;
+    public static final int TODO_START = 5;
+    public static final int EVENT_START = 6;
+    public static final int DEADLINE_START = 9;
+    public static final int DELETE_START = 7;
+    public static final int NEW_TASK_INDEX = 6;
```

Comment:
It is great that you have constants for starting deadline and event start.  However, what happens if there are more than 6 existing tasks?